src/main/java/net/apnic/rdap/filter/filters/RDAPPreFilter.java

```diff
@@ -40,6 +42,8 @@ public Object run()
         RDAPRequestPath path = RDAPRequestPath.createRequestPath(
                 context.getRequest().getRequestURI());
         context.put(RequestContextKeys.RDAP_REQUEST_PATH, path);
+        context.getRequest().setAttribute(ORIGIN_ATT_KEY,
+            context.getRequest().getRequestURL().toString());
```

Comment:
No worries. Just documenting in the code that it is a work around and why it is there is enough I think.